source/change-streams/tests/change-streams.yml

```diff
@@ -297,3 +297,163 @@ tests:
+    operations:
+      -
+        database: *database_name
+        collection: *collection_name
+        name: insertOne
+        arguments:
+          document:
+            x: 1
+      -
+        database: *database_name
+        collection: *collection_name
+        name: updateOne
+        arguments:
+          filter:
+            x: 1
+          update:
+            $set:
+              x: 2
+      -
+        database: *database_name
+        collection: *collection_name
+        name: replaceOne
+        arguments:
+          filter:
+            x: 2
+          replacement:
+            x: 3
+      -
+        database: *database_name
+        collection: *collection_name
+        name: deleteOne
+        arguments:
+          filter:
+            x: 3
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            cursor: {}
+            pipeline:
+              -
+                $changeStream:
+                  fullDocument: default
+          command_name: aggregate
+          database_name: *database_name
+    result:
+      success:
+        -
+          operationType: insert
+          ns:
+            db: *database_name
+            coll: *collection_name
+          fullDocument:
+            x:
+              $numberInt: "1"
+        -
+          operationType: update
+          ns:
+            db: *database_name
+            coll: *collection_name
+          updateDescription:
+            updatedFields:
+              x:
+                $numberInt: "2"
+        -
+          operationType: replace
+          ns:
+            db: *database_name
+            coll: *collection_name
+          fullDocument:
+            x:
+              $numberInt: "3"
+        -
+          operationType: delete
+          ns:
+            db: *database_name
+            coll: *collection_name
+  -
+    description: Test rename and invalidate event types
+    minServerVersion: "4.0.1"
+    target: collection
+    topology:
+      - replicaset
+    changeStreamPipeline: []
+    changeStreamOptions: {}
+    operations:
+      -
+        database: *database_name
+        collection: *collection_name
+        name: rename
+        arguments:
+          to: *collection2_name
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            cursor: {}
+            pipeline:
+              -
+                $changeStream:
+                  fullDocument: default
+          command_name: aggregate
+          database_name: *database_name
+    result:
+      success:
+        -
+          operationType: rename
+          ns:
+            db: *database_name
+            coll: *collection_name
+          to:
+            db: *database_name
+            coll: *collection2_name
+        -
+          operationType: invalidate
+  -
+    description: Test drop and invalidate event types
+    minServerVersion: "4.0.1"
```

Comment:
Unrelated to this PR, but I didn't realize the change stream tests have this at the test-level, instead of file-level as is done in other specs :\

Reply:
@daprahamian: Is this something we can change in the interest of cross-spec consistency? If so, I'll open a spec ticket. It looks like most all of the tests are using `minServerVersion: 3.6.0` and a few use `minServerVersion: 3.8.0`. IMO, those could be split into their own YAML files (the entire spec currently only has two files for error and non-error tests).

Reply:
Opened https://jira.mongodb.org/browse/SPEC-1193 to track this.